projects/ngrx.io/content/guide/store/selectors.md

```diff
@@ -264,7 +264,7 @@ export class AppComponent {
 
 Selectors empower you to compose a [read model for your application state](https://docs.microsoft.com/en-us/azure/architecture/patterns/cqrs#solution).
 In terms of the CQRS architectural pattern, NgRx separates the read model (selectors) from the write model (reducers).
-An advanced technique is to combine selectors with [RxJS pipeable operators](https://github.com/ReactiveX/rxjs/blob/master/doc/pipeable-operators.md).
+An advanced technique is to combine selectors with [RxJS pipeable operators](https://github.com/ReactiveX/rxjs/blob/master/docs_app/content/guide/v6/pipeable-operators.md).
```

Comment:
Thanks for updating the link!
What about pointing to the docs website instead?

```suggestion
An advanced technique is to combine selectors with [RxJS pipeable operators](https://rxjs.dev/guide/v6/pipeable-operators).
```

Reply:
@timdeschryver update link to doc website

Reply:
Great, thanks!